music-playlist-app/index.js

```diff
@@ -0,0 +1,168 @@
+const displayCurrentSong = document.querySelector(".text-container");
+const playlistContainer = document.querySelector(".playlist-container");
+
+// This is the first introduction to getElementById
+const playPauseBtn = document.getElementById("play-btn");
+const previousBtn = document.getElementById("previous-btn");
+const nextBtn = document.getElementById("next-btn");
+const shuffleBtn = document.getElementById("shuffle-btn");
+let isPlaying = false;
+let currentSong = 0;
+
+// This could be a nice review for objects nested inside arrays and how to access those properties
+let playlist = [
+  {
+    id: "We-Are-Going-to-Make-it",
```

Comment:
Filename / id suggestion:

```suggestion
    id: "we-are-going-to-make-it",
```